src/components/LanguageItem/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Image from 'next/image'
+
+interface LanguageItem {
+  readonly locale: string
+  readonly isSelected: boolean
+  readonly handleItemClick: () => void
+}
+
+interface Country {
+  readonly [name: string]: string
+}
+
+const COUNTRY: Country = {
+  en: 'English',
+  th: 'ภาษาไทย',
+}
+
+const LanguageItem = ({ locale, isSelected, handleItemClick }: LanguageItem) => {
+  
+  const renderFlagImage = () => {
+    const imageSize: number = 16
+
+    return locale === 'th' ? (
```

Comment:
Please avoid using magic values.

```suggestion
    return locale === THAI_LOCALE_CODE ? (
```